ts/src/questions/heap/min_heap.ts

```
@@ -9,71 +9,125 @@
-   *
-   */
-  public add(node: T): void {
-    return;
+  public add(node: number): void {
+    this.data.push(node);
+
+    if (this.data.length === 1) {
+      return;
+    }
+
+    return this.bubble_up();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)